assets/lokomotive-kubernetes/packet/flatcar-linux/kubernetes/controllers.tf

```diff
@@ -62,36 +62,21 @@ resource "packet_device" "controllers" {
 }
 
 data "ct_config" "controller-install-ignitions" {
-  count   = var.controller_count
-  content = data.template_file.controller-install[count.index].rendered
-}
-
-data "template_file" "controller-install" {
-  count    = var.controller_count
-  template = file("${path.module}/cl/controller-install.yaml.tmpl")
-
-  vars = {
+  count = var.controller_count
+  content = templatefile("${path.module}/cl/controller-install.yaml.tmpl", {
```

Comment:
why is this changed? The linked issue doesn't say anything either. Is it a simplification at the terraform layer with no effect on the generated config?

Reply:
with tf 0.12, you no longer need `template_file` baked by 3rd party provider, as Terraform provides built-in `templatefile`, so it's recommended ot use this one.

Reply:
Oh, okay. I see in [the docs](https://www.terraform.io/docs/providers/template/d/file.html) that it says so. Linking and explaining in the commit would be nice :)

Reply:
I'll add it, though I'm not fan of duplicating this information, as there will be more commits in the future relating to the same problem. We use GitHub, we should make use of it :smile:

Reply:
Don't know what you mean. The linked issue is not clear to me that upstream recommends not using it anymore.

It's easier to review if you have the context you need. If you prefer to link to the issue only, instead of adding some lines, fine (is very easy to make it easier for reviewers by including those lines, but if you really don't want to do it, don't.). But let's make the issue clear on that, at least.

Reply:
Added a bit more context to both issue and commit message :)

Reply:
Thanks!